docs/source/user/installing.rst

```diff
@@ -71,32 +70,14 @@ To use CUDA with Numba installed by `pip`, you need to install the `CUDA SDK
 installed system-wide on Linux.
 
 
-.. _numba-install-armv7:
-
-Installing on Linux ARMv7 Platforms
------------------------------------
-
-`Berryconda <https://github.com/jjhelmus/berryconda>`_ is a
-conda-based Python distribution for the Raspberry Pi.  We are now uploading
-packages to the ``numba`` channel on Anaconda Cloud for 32-bit little-endian,
-ARMv7-based boards, which currently includes the Raspberry Pi 2 and 3,
-but not the Pi 1 or Zero.  These can be installed using conda from the
-``numba`` channel::
-
-    $ conda install -c numba numba
-
-Berryconda and Numba may work on other Linux-based ARMv7 systems, but this has
-not been tested.
-
-
 Installing on Linux ARMv8 (AArch64) Platforms
 ---------------------------------------------
 
 We build and test conda packages on the `NVIDIA Jetson TX2
 <https://www.nvidia.com/en-us/autonomous-machines/embedded-systems-dev-kits-modules/>`_,
 but they are likely to work for other AArch64 platforms.  (Note that while the
-Raspberry Pi CPU is 64-bit, Raspbian runs it in 32-bit mode, so look at
-:ref:`numba-install-armv7` instead.)
+Raspberry Pi CPU is 64-bit, depending on configuration Raspbian may be running
+it in 32-bit mode).
```

Comment:
The 64-bit RPi OS is "Raspberry Pi OS" (and Raspbian was the 32-bit version, which was also renamed to Raspberry Pi OS). Only certain models are 64-bit. Perhaps the following specifies the situation a little more clearly:

```suggestion
CPUs in the Raspberry Pi 3, 4, and Zero 2 W are 64-bit, Raspberry Pi OS may be
running in 32-bit mode depending on the OS image in use.
```

Reply:
Thanks, added this in 0368ef5, think the suggestion above is missing a `)` on the end, so have added that in too.